### PR TITLE
Work section of portfolio

### PR DESCRIPTION
**Pull request by juaryimami**
In this Project:
Added some pictures
HTML file updated
CSS file updated
README file customized

Thank you for reviewing my project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -39,24 +39,170 @@
...
+						  <li class="works__card__subtitle-li fw-600 fz-13 lh-16 clr-n-500">Back End Dev</li>
+						  <li class="works__card__subtitle-li fw-600 fz-13 lh-16 clr-n-500">2015</li>
+					  </ul>
+				  </div>
+				  <p class="works__card__dsc fw-400 fz-15 lh-24 clr-n-600">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+				  <div class="works__card__topics flex">
+					  <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">html</p>
+					  <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">css</p>
+					  <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">javascript</p>
+				  </div>
```
Comment (cynthiainga) [inline index.html]: - [ ] The list of project technologies are a list `ul` and each of them a `li`, please kindly fix it 😉 😉

File: index.html [hunk 2]
```diff
@@ -39,24 +39,170 @@
...
+						<li class="works__card__subtitle-li fw-600 fz-13 lh-16 clr-n-500">Back End Dev</li>
+						<li class="works__card__subtitle-li fw-600 fz-13 lh-16 clr-n-500">2015</li>
+					</ul>
+				</div>
+				<p class="works__card__dsc fw-400 fz-15 lh-24 clr-n-600">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+				<div class="works__card__topics flex">
+					<p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">html</p>
+					<p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">css</p>
+					<p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">javascript</p>
+				</div>
```
Comment (cynthiainga) [inline index.html:105]: - [ ] The list of project technologies are a list ul and each of them a li, please kindly fix it 😉 😉

File: index.html [hunk 3]
```diff
@@ -39,24 +39,170 @@
...
```
Comment (cynthiainga) [inline index.html:129]: - [ ] Same remark about technologies, the list of project technologies are a list ul and each of them a li, please kindly fix it 😉 😉

File: index.html [hunk 4]
```diff
@@ -39,24 +39,170 @@
...
```
Comment (cynthiainga) [inline index.html:153]: - [ ] - [ ] Same remark about technologies, the list of project technologies are a list ul and each of them a li, please kindly fix it 😉 😉

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @juaryimami, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Good design 👍 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (juaryimami) [reply index.html]: ok thanks, I will change it ASP.

**Review by mugishasam123 — APPROVED** [review]
Hi @juaryimami  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Nice use of git-flow 👍 
- Descriptive PR title 🥇 

## Optional suggestions
[Optional] I would recommend to fix linter errors as they are part of the project and help you keep track of errors regarding code standards

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by juaryimami** [comment]
Thank you so much

**Pull request merged**